jobs/build-arch.Jenkinsfile

```diff
@@ -306,6 +306,15 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             pipeutils.build_artifacts(pipecfg, params.STREAM, basearch)
         }
 
+        // secex specific tests
+        if (shwrapCapture("cosa meta --get-value images.qemu-secex") != "None") {
+            stage("Kola:TestSecex") {
+                // FIXME: Once we have valid HKD for the builder we should add `--qemu-secex-hostkey /path/to/key` switch and enable reboot tests
```

Comment:
I already added the certificate to the builder:
https://github.com/coreos/fedora-coreos-pipeline/issues/833#issuecomment-1494220283
I think we should implement in the test some option to import all certificates from `/data.secex/hostkeys/HKD-*.crt` by default and only take `--qemu-secex-hostkey /path/to/key` as an override.

Reply:
good to know, so now we can mount them `-v /data.secex/hostkeys/key:/secex-hostkey`

Reply:
It is already mounted in the CI by default, since it is a volume.

Reply:
done